llama_stack/providers/remote/inference/groq/groq.py

```diff
@@ -65,7 +65,7 @@
     # TODO(aidand): Replace this with a stable model once Groq supports it
     build_model_alias(
         "llama-3.2-3b-preview",
-        CoreModelId.llama3_2_3b_instruct.value,
+        CoreModelId.llama3_2_3b.value,
```

Comment:
this doesn't look like the right change. most providers don't offer base models by default -- when they offer something, they usually offer the instruct model first because that's what 99% of the customers want.

Reply:
https://console.groq.com/docs/models -- the system card links on Groq's site are definitely pointing wrong. You can see that the 90B vision model is pointing correctly to Instruct, but the 11B vision is not.

Reply:
Interesting. I did not know that they are actually using the instruct one. This is very confusing though. Maybe I should re-purpose this PR to link to their docs?